whylog/assistant/regex_assistant/__init__.py

```diff
@@ -45,10 +45,4 @@ def set_primary_key(self, line_id, primary_key):
         self.regex_matches[line_id].set_primary_key(primary_key)
 
     def validate(self, line_id):
```

Comment:
teacher uses it to take problems related with parser.
look at method `_parser_problems` in teacher whylog/teacher/__init__.py line 182